flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/casting/CastRulesTest.java

```diff
@@ -472,14 +472,12 @@
                                         LocalDateTime.of(2021, 9, 27, 12, 34, 56, 123000000)
                                                 .atZone(CET)
                                                 .toInstant()))
-                        // https://issues.apache.org/jira/browse/FLINK-24446 Fractional seconds are
-                        // lost
                         .fromCase(
                                 STRING(),
                                 CET_CONTEXT,
                                 StringData.fromString("2021-09-27 12:34:56.123456789"),
                                 TimestampData.fromInstant(
-                                        LocalDateTime.of(2021, 9, 27, 12, 34, 56, 0)
+                                        LocalDateTime.of(2021, 9, 28, 22, 52, 32, 789000000)
                                                 .atZone(CET)
```

Comment:
May be a stupid question however could you please clarify why in String value there is 123456789 as fractions of seconds and in target 789000000 ?